recipes/self-managed-otlp-ingest/instrumentation.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  namespace: default
+  name: sample-java-auto-instrumentation
+spec:
+  exporter:
+    endpoint: http://otel-collector:4317
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+  sampler:
+    type: always_on
+
+  java:
+    resources:
+      limits:
+        memory: 512Mi
+      requests:
+        memory: 512Mi
+    env:
+      - name: OTEL_EXPORTER_OTLP_PROTOCOL
+        value: grpc
```

Comment:
Should we just rely on the default value of the SDK?  Any of these should work just fine.

Reply:
Decided to keep it, since the default protocol has been updated in later versions of the collector. The new default is `http/protobuf` so not specifying it explicitly might cause this to stop working if the collector version in `otlp-k8s-ingest` is updated.

Reply:
In the current version of collector used, the default is `grpc`. So omitting it would work with current endpoints with port 4317. 

If the collector version is upgraded (*in otlp-k8s-ingest*), the default will be switched to `http/protobuf` (*in case of unspecified*), but now the current endpoints with port 4317 will not work since http/protobuf will require 4318.

Reply:
Can you try excluding the port from the environment variable?  IIRC, the default port is supposed to be inferred based on the protocol

Reply:
Sounds good, I have updated the instrumentation yaml.

Reply:
Looks like in practice, the port number is required when providing an endpoint.